plugins/inputs/kube_inventory/node.go

```diff
@@ -49,5 +62,37 @@ func (ki *KubernetesInventory) gatherNode(n corev1.Node, acc telegraf.Accumulato
 		}
 	}
 
+	for _, val := range n.Status.Conditions {
+		conditionfields := map[string]interface{}{}
+		conditiontags := map[string]string{}
+		conditiontags["status"] = string(val.Status)
+		conditiontags["condition"] = string(val.Type)
+		for k, v := range tags {
+			conditiontags[k] = v
+		}
+		running := 0
+		nodeready := 0
+		if val.Status == "True" {
+			if val.Type == "Ready" {
+				nodeready = 1
+			}
+			running = 1
+		} else if val.Status == "Unknown" {
+			if val.Type == "Ready" {
+				nodeready = 0
+			}
+			running = 2
```

Comment:
Is there previous knowledge required to know that running 0=notready, 1=ready, 2=unknown? Are these the only values possible currently? Ensure we document this please.

Reply:
Yes the status can be only of three types
```
ConditionTrue    ConditionStatus = "True"
ConditionFalse   ConditionStatus = "False"
ConditionUnknown ConditionStatus = "Unknown"
```
I will add it to the document the number associated with it, since in core v1 it's a string and not a number. 
Just to let you know there are many condition types, but in this code I have only added "Ready" metrics. Below are the other condition types for which we can have similar metrics.
```
MemoryPressure
DiskPressure
PIDPressure
NetworkUnavailable
```